packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -155,14 +155,17 @@ const DetailView = (props) => {
     dividedDetails = details;
   }
 
+  const showTitleSection = (titleElement || secondaryTitlesElements.length || subtitleElements.length || accessoryElement);
```

Comment:
The parentheses are unnecessary.
```suggestion
  const showTitleSection = titleElement || secondaryTitlesElements.length || subtitleElements.length || accessoryElement;
```

Reply:
updated : https://github.com/cerner/terra-clinical/commit/c598ecc9301aad24ee248a1a5a560192d371c448#diff-7c93559c8fcc0937cc491d416ca9b67c7be5123cca0eed7b04d55d11652c5e9d